yata/BulkManager2.py

```diff
@@ -71,13 +71,13 @@ def find_from_batch(self, batch_size=5000):
                     group = '('
                     for f in fields:
                         vals.append(search[f])
-                        group += ' "' + colmap[f] + '" = %s and '
+                        group += ' ' + colmap[f] + ' = %s and '
 
                     group = group.rstrip('and ') + ') or '
                     wherestr += group
 
                 wherestr = wherestr.rstrip('or ')
-                sql = 'select * from "' + self.mgr.model._meta.db_table +'"' + wherestr
```

Comment:
/!\

Reply:
Aren't you suppose to use backticks to quote tables?

I've named tables like `prefix.table_name` and used ` to quote them.